### ISSUE TITLE
Accessibility for the Search Page

### ISSUE THREAD

**Issue by joelbcastillo** [CityOfNewYork/NYCOpenRecords#304]
As a user who is visually impaired, I need to be able to use a screen reader on the search page to navigate the results.

The search results are currently displayed using divs instead of HTML Tables. As a result, screen readers are unable to traverse the results as a table. Using aria-tags will allow screen readers to traverse each "cell", providing a much improved experience for users. 

Acceptance Criteria:
- Use aria tags to allow screen reader to view div rows as tables
- Use aria tags to notify screen reader of updates on the page when search is run.
- Ensure the hierarchy of elements and formatting is consistent (H1 -> P)

An example of using aria to make a table accessible is available on Stack Overflow - https://stackoverflow.com/questions/31008192/using-aria-to-make-a-div-table-accessible
